package.json

```diff
@@ -11,13 +11,12 @@
     "generate": "stencil generate"
   },
   "devDependencies": {
-    "@stencil/core": "^2.0.0",
+    "@stencil/core": "^2.7.0",
```

Comment:
I added this here to show the association with the new version of puppeteer and stencil, since older versions don't necessarily work on all machines (M1 Macs)